### ISSUE TITLE
How to chenge langs?

### ISSUE THREAD

**Issue by 258458** [DualSPHysics/DesignSPHysics#206]

**Comment by luzpaz** [comment]
I don't think this workbench is translated. @imestevez are you interested in translation ? FreeCAD users may be of help here, if you are open to it.

**Comment by imestevez** [comment]
This workbench is not translated and we are not working on it. We are open to receive any contribution on that

**Comment by luzpaz** [comment]
@imestevez we've been slowly translating workbenches  
* https://github.com/FreeCAD/FreeCAD-addons/issues/187  
Would you like to participate in this?

**Comment by luzpaz** [comment]
@imestevez would you be open to receiving a PR from the FreeCAD community ?

cc @hasecilu  any interest ?

**Comment by hasecilu** [comment]
> @luzpaz we decided to add translation to DesignSPHysics. Thank you for the feedback

Are you already into that? Need help?

**Comment by luzpaz** [comment]
@hasecilu I am not. Would you be perhaps be available ?

**Comment by imestevez** [comment]
We are already working on it. I think we can handle it.
I will let you know if anything is needed.
Thank you for your support.

**Comment by luzpaz** [comment]
If you want to leverage our crowdin (3rd party translation software) setup that has a large volunteer translator base, please let us know.

**Comment by luzpaz** [comment]
@imestevez just checking in what the status is here ?
